### ISSUE TITLE
access screensaver settings broken

### ISSUE THREAD

**Issue by jesternl** [cachapa/AerialDream#42]
Hey, seems like it's no longer possible to get into the system screensaver settings from the menu. 
this leaves it unable to set AerialDream as the default screensaver

**Comment by cachapa** [comment]
I'm guessing this is specifically on Google Chromecast with Android TV.

Unfortunately it seems Google decided to remove the screen saver functionality from that device, so there's nothing we can do: https://www.reviewgeek.com/125089/chromecast-with-google-tv-loses-third-party-screensavers

I'll leave this ticket open in case other people come here with the same issue.

**Comment by coyote3** [comment]
I love Aerial Dream so much that I can't live without it.  I see [people talking about](https://stackoverflow.com/questions/73219009/settings-to-set-a-screen-saver-are-not-showing-on-google-chromecast
) [running a couple ADB commands with Android SDK tools](https://www.reddit.com/r/AndroidTV/comments/wkx8x5/chromecast_with_google_tv_loses_thirdparty/
), but the learning curve has daunted me.  I saw (in one of the two pages I just linked to) an Android screensaver developer say he put instructions on his site, anyone willing to do so please?

**Comment by kielolff** [comment]
what would the command be in ADB to set your screensaver?  Something like?

 adb shell settings put secure screensaver_components com.cachapa.aerialdream/.ui.screensaver.DreamActivity

**Comment by JJ2W** [comment]
> I love Aerial Dream so much that I can't live without it. I see [people talking about](https://stackoverflow.com/questions/73219009/settings-to-set-a-screen-saver-are-not-showing-on-google-chromecast) [running a couple ADB commands with Android SDK tools](https://www.reddit.com/r/AndroidTV/comments/wkx8x5/chromecast_with_google_tv_loses_thirdparty/), but the learning curve has daunted me. I saw (in one of the two pages I just linked to) an Android screensaver developer say he put instructions on his site, anyone willing to do so please?

I agree. AerialDream is a must. I have a 2021 Sony TV that plays aerial dream just fine. On AndroidTV 10 OS.
But my 2022 Hisense TV does not. I believe it's running AndroidTV 11 OS.

**Comment by JJ2W** [comment]
@kielolff did you ever find an answer?

**Comment by JJ2W** [comment]
@kielolff @coyote3 @jesternl @cachapa try this::

After you download Aerial Dream onto the TV, setup the settings you like for Aerial Dream in the app.
Once done, ADB into your TV, remotely through Wi-Fi, and then type the following: 
adb shell settings put secure screensaver_components com.codingbuffalo.aerialdream/.AerialDream

**Comment by kielolff** [comment]
> @kielolff @coyote3 @jesternl @cachapa try this::
> 
> After you download Aerial Dream onto the TV, setup the settings you like for Aerial Dream in the app. Once done, ADB into your TV, remotely through Wi-Fi, and then type the following: adb shell settings put secure screensaver_components com.codingbuffalo.aerialdream/.AerialDream


I found "Aerial Views" on github.  the package is the same idea, pulls apple screensavers, developer still updates.  was able to ADB in, install that package and then
"adb shell settings put secure screensaver_components com.neilturner.aerialviews/.ui.screensaver.DreamActivity"

and im back in business.

**Comment by JJ2W** [comment]
> > @kielolff @coyote3 @jesternl @cachapa try this::
> > After you download Aerial Dream onto the TV, setup the settings you like for Aerial Dream in the app. Once done, ADB into your TV, remotely through Wi-Fi, and then type the following: adb shell settings put secure screensaver_components com.codingbuffalo.aerialdream/.AerialDream
> 
> I found "Aerial Views" on github. the package is the same idea, pulls apple screensavers, developer still updates. was able to ADB in, install that package and then "adb shell settings put secure screensaver_components com.neilturner.aerialviews/.ui.screensaver.DreamActivity"
> 
> and im back in business.

That's interesting your ADB command is different than the one I got but glad it works just the same. I guess cause yours is a package and mine is the apk from the play store?
